qmxgraph/common_testing.py

```diff
@@ -32,3 +33,31 @@ def get_cell_ids(widget, filter_function):
         return cells.map(function(aCell){{ return aCell.getId(); }});
     }})()'''.format(filter_func=filter_function))
     return cells_ids
+
+
+@contextmanager
+def bridge_blocker(*bridge_signal_list):
+    """
+    Attach one callback blocker to each signal.
+
+    :param PyQt5.QtCore.pyqtSignal bridge_signal_list: The list of bridge signals.
+    :rtype: List[qmxgraph.callback_blocker.CallbackBlocker]
+    """
+    from qmxgraph.callback_blocker import CallbackBlocker
+    connected_callback_blockers = []
+    try:
+        with ExitStack() as stack:
+            for bridge_signal in bridge_signal_list:
+                cb = CallbackBlocker()
+                bridge_signal.connect(cb)
+                connected_callback_blockers.append((bridge_signal, cb))
+                stack.enter_context(cb)
+
+            yield [cb for _, cb in connected_callback_blockers]
+
+    finally:
+        for bridge_signal, cb in connected_callback_blockers:
+            try:
+                bridge_signal.disconnect(cb)
+            except TypeError:
```

Comment:
Suppress ;)

Reply:
Actually you can use the "safe_disconnect" function in all those cases right?

Reply:
`_silent_disconnect`?

Reply:
yep that's it